frontend/src/components/EntityList.vue

```diff
@@ -155,17 +155,17 @@ export default {
       this.selected = this.$refs.selector.getSelected();
     },
     onDeletion() {
-      const promises = this.selected.map(eId => {
-        this.$api.deleteEntity({
+      const promises = this.selected.map(async eId => {
+        await this.$api.deleteEntity({
           schemaSlug: this.schema.slug,
           entityIdOrSlug: eId
         });
       });
       Promise.all(promises).then(() => this.getEntities({resetPage: true}));
     },
     onRestoration() {
-      const promises = this.selected.map(eId => {
-        this.$api.restoreEntity({
+      const promises = this.selected.map(async eId => {
+        await this.$api.restoreEntity({
```

Comment:
Like above, I think this would restore the intended behavior

```suggestion
      const promises = this.selected.map(eId => {
        return this.$api.restoreEntity({
```